### PR TITLE
Remove extra zero from fomatter #86

### PR DESCRIPTION
**Pull request by modiholodri**
Removed the extra zero ".0" from the axis formatter and change K to lower case k.

image

### CODE REVIEW

File: utils_pandas.py [hunk 1]
```diff
@@ -101,8 +101,8 @@ def human_format(num: float, pos: int) -> str:
         magnitude += 1
         num /= 1000.0
     # add more suffixes if you need them
-    suffix = ['', 'K', 'M', 'G', 'T', 'P'][magnitude]
-    return f'{num:.1f}{suffix}'
+    suffix = ['', 'k', 'M', 'G', 'T', 'P'][magnitude]
```
Comment (reduxionist) [inline utils_pandas.py:104]: is K intentionally made lower case here?

Comment (modiholodri) [reply utils_pandas.py:104]: Yes, I think it should be, but might be that I am wrong and that it is more Austrian style.

Comment (reduxionist) [reply utils_pandas.py:104]: Yes, you are quite correct! Good catch and thanks for the fix! :clap:

**Review by reduxionist — APPROVED** [review]
@djay LGTM

**Review by djay — APPROVED** [review]

**Pull request merged**